exporter.py

```diff
@@ -8,14 +8,19 @@
 import sys
 
 import paho.mqtt.client as mqtt
-from prometheus_client import Gauge, start_http_server
+from prometheus_client import Gauge, Counter, start_http_server
 
 logging.basicConfig(level=os.environ.get("LOG_LEVEL", "INFO"))
 LOG = logging.getLogger("mqtt-exporter")
 PREFIX = os.environ.get("PROMETHEUS_PREFIX", "mqtt_")
 
 # global variable
 prom_metrics = {}  # pylint: disable=C0103
+topic_label = os.environ.get("TOPIC_LABEL", "topic")
+prom_msg_counter = Counter(
+    f"{PREFIX}message_counter", "Counter of received messages", [topic_label]
```

Comment:
according Prometheus best practices, label should end with _total for counters:
https://prometheus.io/docs/practices/naming/

Reply:
Yes makes sense to me however the library seems to some naming magic already:

<img width="1081" alt="image" src="https://user-images.githubusercontent.com/56208/102882512-c0efa200-444e-11eb-9eb7-46caf5a6a743.png">

Reply:
sure, but better explicit than implicit

Reply:
and message_counter_total is kind of redudant